### PR TITLE
feat(ui5-link, ui5-breadcrumbs): add modifiers keys to click event

### PR DESCRIPTION
**Pull request by nnaydenow**
Add modifier keys information to click event.

### CODE REVIEW

**Review by ilhan007 — APPROVED** [review]
There are 3 more components, using the ui5-link inside their templates and we have to change click to ui5-click, but this can be done afterwards in a follow up PR.

File: packages/main/src/Link.js [hunk 1]
```diff
@@ -364,11 +388,12 @@ class Link extends UI5Element {
 			return;
 		}
 
-		event.preventDefault();
+		this._onclick(event);
+
+		if (this.href && !event.defaultPrevented) {
+			const fakeEvent = new MouseEvent("click");
```
Comment (tsanislavgatev) [inline packages/main/src/Link.js:394]: Is fakeEvent the best name for this variable?

**Review by tsanislavgatev — DISMISSED** [review]

File: packages/main/src/Link.js (hunk 1 above)
Comment (nnaydenow) [reply packages/main/src/Link.js:394]: If you have any suggestion, but we create event that will be dispatched through link to simulate click from the link to achieve the default behaviour of link click.

Comment (ilhan007) [reply packages/main/src/Link.js:394]: customEvent for example

**Review by ilhan007 — APPROVED** [review]

**Comment by nnaydenow** [comment]
Hi @hardl,

Just to inform you that this PR is merged.

Regards,
Nayden

**Pull request merged**